src/components/ListOfTodos.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from 'react';
+
+import { TodoComponent } from './TodoComponent';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filterCallback: (todo: Todo) => boolean,
+};
+
+export const ListOfTodos: FC<Props> = ({
+  todos,
+  filterCallback,
+}) => (
+  <section
+    className="todoapp__main"
+  >
+    {todos.filter(filterCallback)
```

Comment:
I would not recommend using `filter` function while rendering. it's better to propagate in `ListOfTodos` component filtered data

Reply:
in `App` component you have `todos` and `filter`. so you can create new variable `filteredTodos` and by using `useMemo` filter data.
it should look like that:
```
const filteredTodos = useMemo(
  () => todos.filter(...),
  [todos, filter],
);
```

Reply:
`filteredTodos` could be used in `ListOfTodos` component

Reply:
Understood. Thx!